kura/org.eclipse.kura.net.admin/src/main/java/org/eclipse/kura/net/admin/modem/quectel/bg96/QuectelBG96.java

```diff
@@ -0,0 +1,375 @@
+/*******************************************************************************
```

Comment:
Please adapt the copyright since it's not a Eurotech contribution

Reply:
Date range should be only 2019 since that is the date of first contribution.

Reply:
Ok done